### PR TITLE
Thu/option to disable out of sync warning prod 13001

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCardHook.js [hunk 1]
```diff
@@ -26,16 +28,23 @@ export const useScenarioDashboardCard = () => {
   );
 
   const isDashboardSync = useMemo(() => {
-    if (!currentScenarioRunStartTime) return true;
+    const disableOutOfSyncWarningBanner = (workspace?.webApp?.options?.disableOutOfSyncWarningBanner ?? false) === true;
```
Comment (esasova) [inline src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCardHook.js:31]: In which case we could need this nullish coalescing operator ? if `disableOutOfSyncWarningBanner` isn't defined in the workspace, `undefined === true` will return false, no ? :thinking:

**Review by esasova — APPROVED** [review]
One question but LGTM :+1:

File: src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCardHook.js (hunk 1 above)
Comment (csm-thu) [reply src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCardHook.js:31]: Oops, you're right, I'll remove it :smile:  Thank you !

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

**Pull request merged**